pubtools/exodus/_tasks/push.py

```diff
@@ -0,0 +1,69 @@
+import logging
+import subprocess
+
+from pubtools.exodus.task import ExodusTask
+
+LOG = logging.getLogger("pubtools-exodus")
+LOG_FORMAT = "%(asctime)s [%(levelname)-8s] %(message)s"
+
+
+class ExodusPushTask(ExodusTask):
+    """Push a directory to the Exodus CDN"""
+
+    def __init__(self):
+        super(ExodusPushTask, self).__init__()
+        self.src = None
+        self.dest = None
+
+    def add_args(self):
+        super(ExodusPushTask, self).add_args()
+
+        self.parser.add_argument(
+            "--source",
+            help="The path to the directory to be pushed to the exodus CDN",
+        )
+        self.parser.add_argument(
+            "--dest",
+            help="The destination path to publish to on the exodus CDN",
+        )
+
+    def run(self):
+        LOG.debug("Exodus push begins")
+
+        if not self.publish:
+            self.publish = self.new_publish()
+            LOG.info("created exodus-gw publish: %s", self.publish["id"])
+
+        self.src = self.args.source
+        self.dest = (
+            self.args.dest
+            if ":" in self.args.dest
+            else "exodus:%s" % self.args.dest
+        )
+
+        cmd = [
+            "exodus-rsync",
+            "--exodus-publish",
+            self.publish["id"],
+            self.src,
+            self.dest,
+        ]
+        LOG.info(cmd)
```

Comment:
I just noticed from the log you showed that this is logging default stringification of a string list.  Could we please explicitly join it instead like ' '.join(cmd) ?

The default formatting for a string list seems not appropriate for end-users.  On python 2 the strings may even be logged as `u`-prefixed string literals which is not something a non-python-developer would necessarily understand.